buildSrc/src/main/groovy/org/opensearch/gradle/precommit/LicenseHeadersTask.groovy

```diff
@@ -77,7 +76,6 @@ class LicenseHeadersTask extends AntTask {
      * constructor can write to it.
      */
     @InputFiles
-    @SkipWhenEmpty
```

Comment:
Never easy, see please https://docs.gradle.org/current/userguide/upgrading_version_7.html#empty_directories_file_tree for the context for this change ...

Reply:
This is not correct. The annotation is not deprecated.The problem is that it is unclear what it means, so it needs to explcitely specify if empty directories are ignored.
With this change you make the task run also for empty inputs.

Reply:
Thanks @uschindler, changing it now